python_modules/dagster/dagster/_utils/test/__init__.py

```diff
@@ -328,3 +328,16 @@ def check_concurrency_claim(
         if not self._sleep_interval:
             return claim_status
         return claim_status.with_sleep_interval(float(self._sleep_interval))
+
+
+def get_all_direct_subclasses_of_marker(marker_interface_cls: Type) -> List[Type]:
```

Comment:
Might want to name this to make it clear that it's looking at Dagster's top-level exports (or allow passing in some object that determines what scope to examine).

Reply:
You have very accurately predicted a change I needed to make in the next PR: https://github.com/dagster-io/dagster/pull/16747/files#r1337902867 👍🏻

Will document it better there.